PROTO_tests/tests/dtypes_test.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+from arkouda import dtypes
+
+"""
+DtypesTest encapsulates arkouda dtypes module methods
+"""
+
+SUPPORTED_NP_DTYPES = [
+    bool,
+    int,
+    float,
+    str,
+    np.bool_,
+    np.int64,
+    np.float64,
+    np.uint8,
+    np.uint64,
+    np.str_,
+]
+
+
+class TestDTypes:
+    @pytest.mark.parametrize("dtype", SUPPORTED_NP_DTYPES)
+    def test_check_np_dtype(self, dtype):
+        dtypes.check_np_dtype(np.dtype(dtype))
+
+    def test_check_np_dtype_errors(self):
+        for dt in np.dtype(np.int16), "np.str", ak.bigint:
+            with pytest.raises(TypeError):
+                dtypes.check_np_dtype(dt)
```

Comment:
we're getting really nit picky, but since this is so small I think I actually like parameterizing this better
```suggestion
    @pytest.mark.parametrize("dtype", [np.dtype(np.int16), "np.str", ak.bigint])
    def test_check_np_dtype_errors(self, dtype):
        with pytest.raises(TypeError):
            dtypes.check_np_dtype(dtype)
```